CMakeLists.txt

```diff
@@ -1,22 +1,65 @@
+cmake_minimum_required(VERSION 3.22)
 PROJECT( twbt )
+set( TWBT_VER "6.xx" )
 
-SET( PROJECT_SRCS twbt.cpp )
-SET( TWBT_VER "5.xx" )
+SET(CMAKE_VERBOSE_MAKEFILE ON)
+if(DEFINED dfhack_SOURCE_DIR)
+    set(DFHACK ${dfhack_SOURCE_DIR})
```

Comment:
Do you need to support out-of-DFHack-tree builds with CMake? It seems to be adding a lot of complexity, which most other plugins avoid by living in the `dfhack/plugins` tree.

Reply:
:3 complexity? It's just a couple if statements

Reply:
I'm referring to all of the logic in this file (not just this branch) that supports out-of-DFHack builds. It's a significant portion of the file.